### ISSUE TITLE
Second passphrase verification

### ISSUE THREAD

**Issue by rigwild** [unik-name/uns-cli#35]
Hello!

I'm trying to create a UNIK token using the `create-unik` command. I get prompted for my main passphrase, but not my second one, making the transaction fail. 
```sh
create-unik -n devnet --explicitValue xxxxx --type organization --verbose
» :info: node: https://forger1.devnet.uns.network;
Enter your wallet passphrase (12 words phrase): ***************************************************************************
Computing UNIK fingerprint... done
unikid: 61bc98d016a173105a91424d0841bb513abffeb0edaa169006dc5b507e23e89d
Creating transaction... done
Transaction id: 65f22f33f650b45eb5c493e49a69f7c44b828b65df0fe1ea7f3ad6ee20619518
Sending transaction... done
» :stop: Transaction not accepted. Caused by: {"65f22f33f650b45eb5c493e49a69f7c44b828b65df0fe1ea7f3ad6ee20619518":[{"type":"ERR_APPLY","message":"[\"Failed to apply transaction, because the second signature could not be verified.\"]"}]};
```

Have a nice day!

**Comment by larry868** [comment]
Hello @rigwild 
Thank you for your feedback. 
This is a missing feature and we're currently working on it. It will be available in the next release, may be late december or more probably begining of january.
Keep in touch.

**Comment by dlecan** [comment]
Fixed by uns-cli v2.2.0.
